src/recipes/cafeconleche.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Café Con Leche - Cuban Style",
+    "description": "The Cuban version of café au lait! A classic Miami breakfast beverage, usually dipped into with a slice of thick buttered toast.",
+    "github": "eprendes",
+    "ingredients": [
+        {
+            "quantity": "1",
+            "measure": "shot (or 1-2 ounces)",
+            "ingredient": "espresso coffee - the Café Bustelo brand is common in Miami"
+        },
+        {
+            "quantity": "1",
+            "measure": "cup",
+            "ingredient": "hot or steamed milk"
+        },
+        {
+            "quantity": "1-3",
+            "measure": "teaspoons",
+            "ingredient": "sugar, optional and to taste"
+        },
```

Comment:
```suggestion
        }
```

Reply:
Thank you! Sorry, this was my first ever json file!